connectors/sources/tests/fixtures/mysql/Makefile

```diff
@@ -2,7 +2,7 @@
 
 PYTHON=python3
 
-export VERSION=8.6.0-SNAPSHOT
+export VERSION=8.6.1-SNAPSHOT
```

Comment:
Not sure if I need to change the version here as well

Reply:
no let's leave it to `0` for now, this is used for pulling elasticsearch, we can use `0` for now